pkg/server/server.go

```diff
@@ -50,6 +50,7 @@ func NewServer(config *viper.Viper) *Server {
 		handlers.NewHandlerBase(zapLogger),
 		config.GetString("OKTA_CLIENT_ID"),
 		config.GetString("OKTA_ISSUER"),
+		!environment.Prod(),
```

Comment:
generally, I don't love this `environment` concept... But I used it here because it is already available, and I didn't want to add more ENV VAR bloat if I don't have to.

Reply:
I think it would be better to explicitly turn this functionality on and off with an ENV var. Another possibility is to make it automatic based on the existence of other env vars (`OKTA_CLIENT_ID`, `OKTA_ISSUER`), but going the explicit route is probably better.

Reply:
Gotcha. Moved to an ENV var.